### ISSUE TITLE
False positive on CommentsIndentationCheck

### ISSUE THREAD

**Issue by cypai** [checkstyle/checkstyle#3054]
Something strange, for some reason it only happens when the lines look like this.

```
package test;

import java.util.ArrayList;

public class Test {
	public void foo() {
		ArrayList<Integer> list = new ArrayList<>();
		// foo

		// bar
		list.size();
		// baz
	}
}
```

Message is: Comments Indentation: Comment has incorrect indentation level 2, expected is 11, indentation should be the same level as line 11.

**Comment by romani** [comment]
@cypai ,
please report issue as described here - http://checkstyle.sourceforge.net/report_issue.html#How_to_report_a_bug

**Comment by cypai** [comment]
```
~/work/test$ javac Test.java 
~/work/test$ cat Test.java 
package test;

import java.util.ArrayList;

public class Test {
	public void foo() {
		ArrayList<Integer> list = new ArrayList<>();
		// foo

		// bar
		list.size();
		// baz
	}
}
~/work/test$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="CommentsIndentationCheck"/>
    </module>
</module>
~/work/test$ java -jar checkstyle-6.16.1-all.jar -c config.xml Test.java
Starting audit...
[ERROR] /home/cpai/work/test/Test.java:12: Comment has incorrect indentation level 2, expected is 11, indentation should be the same level as line 11. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
```

**Comment by MEZk** [comment]
@romani 
Smae problem https://github.com/checkstyle/checkstyle/issues/2998
